### PR TITLE
Packetloss and lsyncd (sync coredumps)

### PR DESCRIPTION
**Pull request by alevkin**

### CODE REVIEW

File: tank/resources/ansible/core.yml [hunk 1]
```diff
@@ -99,3 +101,17 @@
...
+    - name: Install packetloss script
+      template:
+        src: templates/set-packetloss.j2
+        dest: "/usr/local/bin/set-packetloss"
+        mode: 0755
+    - name: Schedule applying iptables rules
+      at:
+        command: "/usr/local/bin/set-packetloss add {{ tank['packetloss'][ansible_local['tank_meta']['region']] | default('0.0') }}"
+        count: 3
+        units: minutes
```
Comment (Eenae) [inline tank/resources/ansible/core.yml]: Decided to set packetloss for TCP 1024+

File: tank/resources/ansible/core.yml [hunk 2]
```diff
@@ -99,3 +101,17 @@
...
+  hosts: "all"
+  tasks:
+    - name: Install packetloss script
+      template:
+        src: templates/set-packetloss.j2
+        dest: "/usr/local/bin/set-packetloss"
+        mode: 0755
+    - name: Schedule applying iptables rules
+      at:
+        command: "/usr/local/bin/set-packetloss add {{ tank['packetloss'][ansible_local['tank_meta']['region']] | default('0.0') }}"
```
Comment (Eenae) [inline tank/resources/ansible/core.yml]: Decided to let the python part handle setting calculation per resource. `tank['packetloss']` should be a ready-made number.

File: tank/resources/providers/digitalocean/main.tf [hunk 3]
```diff
@@ -3,13 +3,16 @@
...
 variable "ssh_fingerprint" {}
+variable "scripts_path" {}
 
 # test case-specific settings
 variable "blockchain_name" {}
 
 # run-specific settings
 variable "setup_id" {}
-
+variable "tank_region" {
```
Comment (Eenae) [inline tank/resources/providers/digitalocean/main.tf]: Please, don't introduce the variable. The region values should be passed via templating, not variables.

File: tank/resources/scripts/tank-packetloss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+function delete(){
+  _current_packetloss_rules_input=$(iptables -L INPUT --line-numbers | grep "TANK: packetloss"|awk '{print $1}')
+  _current_packetloss_rules_output=$(iptables -L OUTPUT --line-numbers | grep "TANK: packetloss"|awk '{print $1}')
+  iptables -D INPUT ${_current_packetloss_rules_input}
+  iptables -D OUTPUT ${_current_packetloss_rules_output}
+
+}
+
+$1
```
Comment (Eenae) [inline tank/resources/scripts/tank-packetloss]: an explicit check needed - `add` or `delete`. Refuse to exec any other value

Comment (alevkin) [reply tank/resources/scripts/tank-packetloss]: Done

**Pull request merged**